docs/usage.md

```diff
@@ -42,23 +42,12 @@ The Rust library crate provides the following capabilities:
 * `no_interleaved_io` forces fully-synchronous I/O; otherwise, the library uses threaded I/O for some operations to improve performance.
 * `fetch_remote_manifests` enables the verification step to retrieve externally referenced manifest stores.  External manifests are only fetched if there is no embedded manifest store and no locally adjacent .c2pa manifest store file of the same name.
 * `json_schema` is used by `make schema` to produce a JSON schema document that represents the `ManifestStore` data structures.
-* `openssl_ffi_mutex` prevents multiple threads from accessing the C OpenSSL library simultaneously. (This library is not re-entrant.) In a multi-threaded process (such as Cargo's test runner), this can lead to unpredictable behavior.
 
 ### New API
 
-### Enabling 
+The new api is now enabled by default, the `unstable_api` feature is no longer used.
 
-<!-- This requirement should go away with actual 1.0 release -->
-
-The current release has a new API that replaces the previous methods of reading and writing C2PA data, which are still supported but will be deprecated.  
-
-To use the new API, enable the `unstable_api` feature; for example:
-
-```
-c2pa = {version="0.39.0", features=["unstable_api"]}
-```
-
-When version 1.0 of the library is released, the new API will become the default, but you will still be able to use the deprecated API by enabling the `v1_api` feature; for example:
+You are still be able to use the deprecated API by enabling the `v1_api` feature; for example:
```

Comment:
```suggestion
You will still be able to use the deprecated API by enabling the `v1_api` feature; for example:
```

Reply:
I'd prefer: 
To use the deprecated v1 API, enable the `v1_api` feature; for example:

Reply:
going with Rand's recommendation